### DISCUSSION TITLE
python figure output path delimited by '\' on windows

### DISCUSSION THREAD

**Discussion by Yoshinobu-Ishizaki** [quarto-dev/quarto-cli#894] [Q&A] [answered]
Thank you for developing this fantastic tool!

When I render using VSCode or RStudio on Windows , '.qmd' file below produces figure path like, 

```
![]($(input-filename)_files\figure-gfm/cell-2-output-1.png)
```

This is not desirable when I want to put output files onto web server.
'\\' should be '/' for platform independent output.

How can I fix this? (Could not figure out source of this behavior...) 

Source example.
Whether the format is "gfm" or "html", delimiter behavior is same.

````
---
title: ""
jupyter: python3
format: gfm
---

```{python}
#  
import matplotlib.pyplot as plt
import numpy as np 

x = np.linspace(1,5,100)
y = x**2

plt.plot(x,y)
```
````

With RStudio + R, it is OK ( only '/' is used to form link to figure). 

---

Quarto 0.9.402 on Windows 10 , VSCode 1.67.1,  RStudio 2022-02, Python 3.10.0

**Comment by jjallaire** [comment, marked as answer]
Thanks for reporting this! Should be fixed here: https://github.com/quarto-dev/quarto-cli/commit/63587284be9952fa7a8493347bfb9197e4c977c0

**Reply by Yoshinobu-Ishizaki** [reply]
Thank you for the quickest fix ! ❤️
